cli/buffcli/src/bin/buff/settings.rs

```diff
@@ -0,0 +1,39 @@
+use std::env;
```

Comment:
what does settings represents? i think we need to rethink naming/objects/contexts here..